app/handlers/models.py

```diff
@@ -23,10 +23,15 @@
 
 @api.route("")
 class ModelList(Resource):
-
-    def get(self) -> Dict[str, Any]:
+    #@api.param("model_id", description="The model ID")
+    @api.marshal_with(model_metadata, code=200)
+    @api.response(400, "Invalid input")
+    def get(self, model_id: int) -> Dict[str, Any]:
```

Comment:
I think `model_id` should be removed so that the API endpoint can fetch a list of all the model metadata.